node/src/actors/chain_manager/mod.rs

```diff
@@ -2336,15 +2338,15 @@ pub fn log_removed_transactions(removed_transactions: &[Transaction], inserted_t
 
         if removed_the_one_we_just_inserted {
             log::trace!(
-                "Transaction {} was not added because the TransactionsPool is full",
+                "Transaction {} was not added because the fee was too low",
```

Comment:
This is tricky, because now there are 2 reasons why a transaction is not inserted.

```suggestion
                "Transaction {} was not added",
```

Reply:
I changed the comment to `the fee was too low` because this is essentially also what happens when you do not add the new transaction because of the size limitations to the transaction pool. If the fee of said transaction was high enough, it would've kicked out another transaction instead of not adding the new one. Or do I misunderstand something?

Reply:
True, good idea